msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/AccountState.kt

```diff
@@ -77,7 +77,7 @@ class AccountState private constructor(
 
     constructor (parcel: Parcel) : this (
         account = parcel.serializable<IAccount>() as IAccount,
-        correlationId = parcel.readString() ?: "UNSET",
+        correlationId = parcel.readString() ?: "",
```

Comment:
Doesn't `UUID.fromString("")` also produce an exception?

Reply:
That's a good point. How about generating random UUID here?

Reply:
Keep the original "UNSET" string.